lib/game.rb

```diff
@@ -0,0 +1,82 @@
+require 'pry'
+
+class Game
+  WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8],
+  [0,3,6], [1,4,7], [2,5,8],
+  [0,4,8], [6,4,2]]
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+  def current_player
+
```

Comment:
Consider using a ternary operator for this method

`@board.turn_count.even? ? @player_1 : @player_2`